src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -562,23 +570,49 @@
       {% set label = name|humanize %}
     {% endif %}
 
-    {% if material_design is defined or attr.material_design is defined %}
+    {% if block_prefixes[2] == 'radio' %}
+      <label class="form-check-label"{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
+        {{- widget|raw -}}
+
+        <i class="form-check-round"></i>
```

Comment:
Can this have side-effects on other pages than Product?

Reply:
Yes of course, it makes the BO fully using material checkboxes (at least components using prestashop_ui_kit_base, I was discussing with @jolelievre and @PierreRambaud when I did this, it looks like it's a good idea as the Product team always as us to switch to material design.

Also, we shouldn't considered it as a BC Break as there is a checkbox inside the markup, so everything should be fine